### ISSUE TITLE
Can't get an wildcard certificate

### ISSUE THREAD

**Issue by kostich** [robin-thoni/certbot-pdns#2]
Using certbot-pdns 1.1.0 installed via pip3. Can't get an wildcard certificate:

```
# certbot --agree-tos --text --renew-by-default --server https://acme-v02.api.letsencrypt.org/directory --authenticator certbot-pdns:auth -m support@something.eu -d *.sub.domain.eu
Saving debug log to /var/log/letsencrypt/letsencrypt.log

Plugins selected: Authenticator certbot-pdns:auth, Installer apache
/usr/local/lib/python3.5/dist-packages/josepy/jwa.py:107: CryptographyDeprecationWarning: signer and verifier have been deprecated. Please use sign and verify instead.
  signer = key.signer(self.padding, self.hash)
Performing the following challenges:
dns-01 challenge for sub.domain.eu
Cleaning up challenges
An unexpected error occurred:
TypeError: Can't convert 'bytes' object to str implicitly
Please see the logfiles in /var/log/letsencrypt for more details.
```

**Comment by robin-thoni** [comment]
I think your certbot is using python3 instead of python2. This plugin is intended to be used with python2. Can you check this?

**Comment by kostich** [comment]
I am indeed using Python3. I've created a PR which fixes this issue. I didn't notice any other issues when using certbot-pdns and Python 3.

**Comment by robin-thoni** [comment]
I merged your PR and updated the pip package to v1.1.1

Thanks!
